src/battle-animations.ts

```diff
@@ -714,16 +720,16 @@ class BattleScene {
 	}
 
 	teamPreviewEnd() {
-		for (let siden = 0; siden < 2; siden++) {
-			this.$sprites[siden].empty();
+		for (let siden = 0; siden < this.battle.sides.length; siden++) {
+			this.$sprites[siden % 2].empty();
 			this.battle.sides[siden].updateSprites();
 		}
```

Comment:
`siden % 2` isn't `spriteIndex`, right? It might be easier if we did these in separate loops.

```suggestion
		this.$sprites[0].empty();
		this.$sprites[1].empty();
		for (let siden = 0; siden < this.battle.sides.length; siden++) {
			this.battle.sides[siden].updateSprites();
		}
```

Reply:
(I'll handle this)